Tools/Scripts/run-jsc-stress-tests

```diff
@@ -1955,6 +1955,10 @@ def runWebAssemblySIMDSpecTest(mode)
     runWebAssemblySpecTestBase(mode, "simd-spec-harness", "-eager", "--useWebAssemblySIMD=true", "--useBBQJIT=1", "--useWasmLLInt=1", "--wasmLLIntTiersUpToBBQ=1", "--thresholdForOMGOptimizeAfterWarmUp=0", "--thresholdForOMGOptimizeSoon=0")
 end
 
+def runWebAssemblyTailCallSpecTest(mode)
+    runWebAssemblySpecTestBase(mode, "tail-call-spec-harness", nil, "--useWebAssemblyTailCalls=true")
```

Comment:
this is the only change since last push.
https://github.com/WebKit/WebKit/commit/7848e21c20b3d957698bb3bb376511250c38a3b5 introduced a third `postfix` parameter  to `runWebAssemblySpecTestBase`, which this call didn't handle.